recipe/meta.yaml

```diff
@@ -11,9 +11,10 @@ source:
     sha256: 47a5433fcdd639db836b99e1b5459c2b813cbdad23ff2b5dd4ad27f792ba918e
     patches:
       - 0001-Allow-flags-other-than-macros-in-CPPFLAGS.patch
+      - 0024-Link-to-absl_spinlock_wait.patch.txt
```

Comment:
Is the `.txt` extension correct for our patches?

Reply:
Good catch, corrected.